docs/event-listener.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Gravitino event listener"
+slug: /event-listener
+date: 2024-04-18
+keyword: event listener
+license: Copyright 2024 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+## Introduction
+
+Gravitino provides event listener mechanism to allow use to capture the event provides by Gravitino server to inject some custom operations.
```

Comment:
use -> users?

Reply:
What's the meaning of `inject some custom operations`?

Reply:
rewrited

Reply:
> use -> users?

You haven't resolved it.

Reply:
`to allow uses to capture the event`, please check it again